generated/niswitch/niswitch.proto

```diff
@@ -1,6 +1,6 @@
 
 //---------------------------------------------------------------------
-// This file is automatically generated. All manual edits will be lost.
+// This file is generated from NI-SWITCH API metadata version 22.8.0d130
```

Comment:
I think we'll need to revert updating the comment for now. You can just use the generated .proto file here instead of the one from hapigen.

I have this [Task](https://dev.azure.com/ni/DevCentral/_workitems/edit/2128588) to add validation against source proto files and what's generated in the generated folder. I think I'll also need to add this updated comment generation to the proto gen in grpc_device for all the .proto files. I can do that in a separate PR though.

Reply:
Pushed a commit for what is built locally rather than what hapigen exported. This should pass the prebuild now.